src/registrar/models/user.py

```diff
@@ -17,6 +17,18 @@ class User(AbstractUser):
     but can be customized later.
     """
 
+    # #### Constants for choice fields ####
+    INELIGIBLE = "ineligible"
+    STATUS_CHOICES = ((INELIGIBLE, INELIGIBLE),)
```

Comment:
Dumb question, but why is INELIGIBLE set twice as a tuple? 😅

Reply:
Not a dumb question as all. I just duplicated what I found in domain_application, assuming that we're setting the INELIGIBLE string defined above to an INELIGIBLE property. I may be wrong.

Reply:
The link and explanation helps a lot, thank you so much @neilmb!